<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EEG信号采集系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="serialTab">
       <attribute name="title">
        <string>串口设置</string>
       </attribute>
       <layout class="QVBoxLayout" name="serialTabLayout">
        <item>
         <widget class="QWidget" name="controlPanel" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QGroupBox" name="receiverSerialGroup">
             <property name="title">
              <string>接收端串口设置</string>
             </property>
             <layout class="QFormLayout" name="formLayout">
              <item row="0" column="0">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>串口:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="portCombo"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>波特率:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="baudrateCombo">
                <item>
                 <property name="text">
                  <string>9600</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>19200</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>38400</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>57600</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>115200</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>230400</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>460800</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>数据位:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="databitsCombo">
                <item>
                 <property name="text">
                  <string>5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>6</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>7</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>8</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>校验位:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="parityCombo">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Odd</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Even</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Mark</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Space</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_5">
                <property name="text">
                 <string>停止位:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QComboBox" name="stopbitsCombo">
                <item>
                 <property name="text">
                  <string>1</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>1.5</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>2</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="5" column="0">
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>流控:</string>
                </property>
               </widget>
              </item>
              <item row="5" column="1">
               <widget class="QComboBox" name="flowcontrolCombo">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>XON/XOFF</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>RTS/CTS</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>DSR/DTR</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="receiverButtonGroup">
             <property name="title">
              <string>接收端控制</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPushButton" name="openCloseButton">
                <property name="text">
                 <string>打开串口</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="refreshButton">
                <property name="text">
                 <string>刷新端口</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="clearButton">
                <property name="text">
                 <string>清空接收区</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="tgamSerialGroup">
             <property name="title">
              <string>TGAM 信号串口设置</string>
             </property>
             <layout class="QFormLayout" name="formLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_8">
                <property name="text">
                 <string>串口:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QComboBox" name="tgamPortCombo"/>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_9">
                <property name="text">
                 <string>波特率:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QComboBox" name="tgamBaudrateCombo">
                <item>
                 <property name="text">
                  <string>9600</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>19200</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>38400</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>57600</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>115200</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_10">
                <property name="text">
                 <string>数据位:</string>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QComboBox" name="tgamDatabitsCombo">
                <item>
                 <property name="text">
                  <string>8</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_11">
                <property name="text">
                 <string>校验位:</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QComboBox" name="tgamParityCombo">
                <item>
                 <property name="text">
                  <string>None</string>
                 </property>
                </item>
               </widget>
              </item>
              <item row="4" column="0">
               <widget class="QLabel" name="label_12">
                <property name="text">
                 <string>停止位:</string>
                </property>
               </widget>
              </item>
              <item row="4" column="1">
               <widget class="QComboBox" name="tgamStopbitsCombo">
                <item>
                 <property name="text">
                  <string>1</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QGroupBox" name="tgamButtonGroup">
             <property name="title">
              <string>TGAM 信号控制</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QPushButton" name="tgamOpenCloseButton">
                <property name="text">
                 <string>打开TGAM串口</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="tgamRefreshButton">
                <property name="text">
                 <string>刷新TGAM端口</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="toggleSimButton">
                <property name="text">
                 <string>开始模拟</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="saveDataButton">
                <property name="text">
                 <string>保存数据</string>
                </property>
               </widget>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QWidget" name="statusStatsWidget" native="true">
           <layout class="QHBoxLayout" name="horizontalLayout_3">
            <item>
             <widget class="QLabel" name="statusLabel">
              <property name="styleSheet">
               <string notr="true">color: red;</string>
              </property>
              <property name="text">
               <string>接收端串口状态: 关闭</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="tgamStatusLabel">
              <property name="styleSheet">
               <string notr="true">color: red;</string>
              </property>
              <property name="text">
               <string>TGAM串口状态: 关闭</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="receivedStatsLabel">
              <property name="text">
               <string>接收字节: 0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QGroupBox" name="receiveSettingsGroup">
              <property name="title">
               <string>接收设置</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="text">
                  <string>显示格式:</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QComboBox" name="asciiRadio">
                 <item>
                  <property name="text">
                   <string>ASCII</string>
                  </property>
                 </item>
                 <item>
                  <property name="text">
                   <string>Hex</string>
                  </property>
                 </item>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="autoWrapCheck">
                 <property name="text">
                  <string>自动换行</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QCheckBox" name="showTimeCheck">
                 <property name="text">
                  <string>显示时间戳</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="receiveWidget" native="true">
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QGroupBox" name="receiveGroup">
              <property name="title">
               <string>EEG数据接收</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QTextEdit" name="receiveText">
                 <property name="lineWrapMode">
                  <enum>QTextEdit::NoWrap</enum>
                 </property>
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="plotTab">
       <attribute name="title">
        <string>实时绘图</string>
       </attribute>
       <layout class="QVBoxLayout" name="plotTabLayout">
        <item>
         <widget class="QWidget" name="plotWidget" native="true">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>400</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionRefresh">
   <property name="text">
    <string>刷新接收端端口</string>
   </property>
  </action>
  <action name="actionRefreshTGAM">
   <property name="text">
    <string>刷新TGAM端口</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
